wp/lib/bap_wp/Makefile

```diff
@@ -14,7 +14,7 @@ install.local: build
 	dune install
 	opam pin add -y .
 
-test: test.unit
+test: test.unit test.performance
```

Comment:
Didn't we remove this for a reason?

Reply:
We removed it because the slowdown was coming from building the string rather than Z3. I fixed that, so that isn't the case anymore.

I put it back because we had some changes that broke the performance tests, and I think it would be a good idea to keep track of it even if we don't have that sophisticated of a test case.

Reply:
Did they break because of performance regression?

Reply:
If not, might as well add a faster version to the "ordinary" tests...

Reply:
No, it was just an API change that caused the performance tests to be unable to compile